config/initializers/date.rb

```diff
@@ -0,0 +1,6 @@
+# Extends Date class with convenience methods
+class Date
+  def display_date
```

Comment:
How does this interact with the Time definition of the same? I'm assuming it doesn't

https://github.com/DCAFEngineering/dcaf_case_management/blob/e4eeca3aef354644d2d1a8b101aff3e484ac91a8/config/initializers/time.rb#L3-L5